config/initializers/modify_rails_form_builder.rb

```diff
@@ -0,0 +1,14 @@
+class ActionView::Helpers::FormBuilder
+  def error_message_for(field_name)
```

Comment:
I'm with Gray for this one. I'm really not a fan of monkey patching. If it does need to happen you could consider using `class_eval` or  `Module#prepend/extend` to inject behaviour, but that's really only slightly better that opening up the class directly.

I think in this case there is no reason for this not to be a view helper which accepts an object and a field name and renders the errors.